administrator/components/com_joomgallery/helpers/upload.php

```diff
@@ -643,11 +644,7 @@ protected function uploadBatch()
       // because the uploaded archive is saved like php8900.tmp and JArchive
       // needs a valid extension
       $zipfile = $extractdir.'/'.$zippack['name'];
-      if(!JFile::upload($zippack['tmp_name'], $zipfile, false, true))
```

Comment:
With this change you are reverting changes of #24. Is that intended?

Reply:
No, that was not intended. Thanks...

Reply:
Thanks! Hope that there were no other code changes missing in the branch you started this pull request with 😨 .

Reply:
I started the PR with the v3.5.0-Branch soon after release of v3.4.0 ...

Reply:
But you did not start with this branch https://github.com/JoomGalleryfriends/JoomGallery/commits/v3.5.0 . You created your own branch https://github.com/Elfangor93/JoomGallery/commits/v3.5.0 , which has one commit more namely https://github.com/Elfangor93/JoomGallery/pull/4 , a pull request you created by yourself. You should have used https://github.com/JoomGalleryfriends/JoomGallery/commits/v3.5.0 .

Reply:
You are right...
But the change you mentioned here was correct in my own v3.5.0. Therefore the wrong code must have sneaked in during my changes. The used branch (https://github.com/Elfangor93/JoomGallery/commits/v3.5.0) was at the same state as the original v3.5.0 (https://github.com/JoomGalleryfriends/JoomGallery/commits/v3.5.0).

Reply:
However it happened, hopefully it was the only wrong code that must have sneaked in during your changes 😉  😄.

Reply:
I did another check:
As far as I can see, all PRs up to #28 are included in this PR. All PRs merged in branch v3.4.1 (those above #30) are not included.

Reply:
A branch v3.4.1 does not exist!? The master branch is some commits ahead to the branch v3.5.0 namely with some pull requests belonging to the milestone 3.4.1 (bug fixes). They can be merged to the v3.5.0 branch any time later. Or maybe we will merge v3.5.0 to the master branch later if the next release will not be v3.4.1 but v3.5.0!? What is the problem with that? I don't see what this has to do with the problem we discussed above?

Reply:
No, there is no problem with that. I just wanted to be sure, that I dont revert any previous PR...

Reply:
Sorry, but I didn't get what you were going to tell me with the last comment.

Reply:
Another try:
I am sure now after my additional checks, that my PR includes all changes up to #28.

Reply:
That sounds good 😄 . Thank you!